app/src/main/java/co/nimblehq/compose/crypto/ui/theme/AppColors.kt

```diff
@@ -0,0 +1,65 @@
+package co.nimblehq.compose.crypto.ui.theme
+
+import androidx.compose.material.*
+import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.ui.graphics.Color
+
+/**
+ * Extend final [Colors] class to provide more custom app colors.
+ */
+data class AppColors(
```

Comment:
Should we make it internal data class 🤔 
```suggestion
internal data class AppColors(
```

Reply:
@Wadeewee makes sense 👍

Reply:
I don't think we should mark these classes as internal. We use `internal` in some local vals in the theme package to show that they are local vars for app theme config and they should not be used directly in the production code, such as `LocalColors.coinNameText` ❌